llvm/lib/Target/RISCV/RISCVInstrInfoXSf.td

```diff
@@ -450,6 +474,18 @@ multiclass VPatVC_XVV<string intrinsic_suffix, string instruction_suffix,
                      "PseudoVC_V_" # instruction_suffix # "_" # vti.LMul.MX,
                      wti.Vector, wti.Vector, vti.Vector, type, vti.Log2SEW,
                      wti.RegClass, vti.RegClass, kind, op1_kind>;
+  // Add another patterns for float type return value.
+  if !ne(wti.SEW, 8) then {
+    defvar wfti = !cast<VTypeInfo>("VF"#wti.SEW#wti.LMul.MX);
```

Comment:
Do we need new tests for this?

Reply:
Yes, I'll add some tests for it~

Reply:
Do you think test cases all of combination of vector type arguments(including float and int) are needed?
Or just provide test cases for some mix type arguments are enough.

Reply:
some mix is probably good enough